media-gfx/graphviz/graphviz-2.40.1.recipe

```diff
@@ -0,0 +1,165 @@
+SUMMARY="Open source graph visualization software"
+DESCRIPTION="Graphviz is open source graph visualization software. Graph \
+visualization is a way of representing structural information as diagrams of \
+abstract graphs and networks. It has important applications in networking, \
+bioinformatics,  software engineering, database and web design, machine \
+learning, and in visual interfaces for other technical domains.
+The Graphviz layout programs take descriptions of graphs in a simple text \
+language, and make diagrams in useful formats, such as images and SVG for web \
+pages, PDF or Postscript for inclusion in other documents; or display in an \
+interactive graph browser. (Graphviz also supports GXL, an XML dialect.)
+Graphviz has many useful features for concrete diagrams, such as options for \
+colors, fonts, tabular node layouts, line styles, hyperlinks, rolland custom \
+shapes."
+HOMEPAGE="http://www.graphviz.org/"
+COPYRIGHT="1994-2010 AT&T Corp."
+LICENSE="CPL v1.0"
+REVISION="1"
+SOURCE_URI="http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz.tar.gz"
+CHECKSUM_SHA256="ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421"
+SOURCE_FILENAME="graphviz-$portVersion.tar.gz"
+PATCHES="graphviz-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libcdtVersion="5.0.0"
+libcgraphVersion="6.0.0"
+libgvcVersion="6.0.0"
+libgvprVersion="2.0.0"
+liblab_gamutVersion="1.0.0"
+libpathplanVersion="4.0.0"
+libxdotVersion="4.0.0"
+libcdtVersionCompat="$libcdtVersion compat >= ${libcdtVersion%%.*}"
+libcgraphVersionCompat="$libcgraphVersion compat >= ${libcgraphVersion%%.*}"
+libgvcVersionCompat="$libgvcVersion compat >= ${libgvcVersion%%.*}"
+libgvprVersionCompat="$libgvprVersion compat >= ${libgvprVersion%%.*}"
+liblab_gamutVersionCompat="$liblab_gamutVersion compat >= ${liblab_gamutVersion%%.*}"
+libpathplanVersionCompat="$libpathplanVersion compat >= ${libpathplanVersion%%.*}"
+libxdotVersionCompat="$libxdotVersion compat >= ${libxdotVersion%%.*}"
+
+PROVIDES="
+	graphviz$secondaryArchSuffix = $portVersion compat >= 2
+	cmd:acyclic$secondaryArchSuffix
+	cmd:bcomps$secondaryArchSuffix
+	cmd:ccomps$secondaryArchSuffix
+	cmd:circo$secondaryArchSuffix
+	cmd:cluster$secondaryArchSuffix
+	#cmd:diffimg$secondaryArchSuffix
```

Comment:
```diff
-	#cmd:diffimg$secondaryArchSuffix

Reply:
These commands don't seem to be part of the package?

Reply:
My bad, misread these here (those can be removed yes) ...